flake.nix

```diff
@@ -94,157 +92,166 @@
           '';
         };
 
+        # Custom packages and aliases for building hosts
+        # Accessible through 'nix build', 'nix run', etc
         packages = with flake.nixosConfigurations; {
-          "homestakeros" = homestakeros.config.system.build.kexecTree;
+          "homestaker" = homestaker.config.system.build.kexecTree;
```

Comment:
I think this should be reverted, because homestakeros is googleable, homestaker is not. Another option could also do homestakerOS.

Reply:
You're right, I was in two minds about this. I will revert this.